app/v3/httpTriggerSqlInput/function.json

```diff
@@ -17,7 +17,7 @@
             "name": "inputItem",
             "direction": "in",
             "connectionStringSetting": "e2eTest_sql",
-            "commandText": "select * from dbo.e2eTestTable where id = @id",
+            "commandText": "select * from dbo.e2eSqlNonTriggerTable where id = @id",
```

Comment:
Where can I find this table name (for future reference)?

Reply:
The tables get created in this file: https://github.com/Azure/azure-functions-nodejs-e2e-tests/pull/36/files#diff-5d7c8d0e102e8d95a27475f9eff5bf8390bfa38648693f61081983b8188b2042R23

Assuming you run the script to create the resources, that means you ideally shouldn't have to remember the name